random_question/weighted_random.py

```diff
@@ -1,6 +1,6 @@
 from collections import defaultdict
-
```

Comment:
There is a very strict way of how to format your code. For example, there should be a constant amount of lines between the `import`s and the actualy code.

No worries!
Pressing `ctrl+alt+l` in pycharm should auto-format your code (so always always press it)